Breeze.UI/src/app/wallet/send/send.component.ts

```diff
@@ -230,32 +235,31 @@ export class SendComponent implements OnInit {
   };
 
   private sendTransaction(hex: string) {
-    let transaction = new TransactionSending(hex);
+    const transaction = new TransactionSending(hex);
     this.apiService
       .sendTransaction(transaction)
       .subscribe(
         response => {
-          if (response.status >= 200 && response.status < 400){
-            this.activeModal.close("Close clicked");
+          if (response.status >= 200 && response.status < 400) {
+            this.activeModal.close('Close clicked');
           }
         },
         error => {
           console.log(error);
           this.isSending = false;
           if (error.status === 0) {
-            //this.genericModalService.openModal(null, null);
-            this.apiError = "Something went wrong while connecting to the API. Please restart the application."
+            // this.genericModalService.openModal(null, null);
+            this.apiError = 'Something went wrong while connecting to the API. Please restart the application.'
           } else if (error.status >= 400) {
             if (!error.json().errors[0]) {
               console.log(error);
-            }
-            else {
-              //this.genericModalService.openModal(null, error.json().errors[0].message);
+            } else {
+              // this.genericModalService.openModal(null, error.json().errors[0].message);
```

Comment:
does `openModal` still take 2 parameters for options? I thought not

Reply:
It is commented out. I wasn't sure whether to keep these comments or not. Generally, I would advice against having any commented out for that specific reason as it is hard for others to know whether it is needed or not

Reply:
`error.json().errors[0].message` is returned and then used in an error. I think that's why that comment is there, to explain why the return is the way it is. Since your handler no longer takes messages maybe it should return `{body: error.json().errors[0].message }`

Reply:
Looking at the code it looks like it used to popup this error and now it just populates a message `apiError` which is bound to a div. I think it is safe to remove this commented out code. `apiError` is not actually returned anywhere.